frontend/packages/console-metal3/src/components/machine-cell.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Icon } from 'patternfly-react';
+import {
+  canHostAddMachine,
+  DASHES,
```

Comment:
Nit: isn't there some constant that we can import from `@console/internal` instead?

Reply:
It seems that simple '-' string is used frequently in the codebase.